### PR TITLE
Issue#194 Make maven log less verbose on CI build

### LINKED ISSUE #194: Make maven log less verbose on CI builds
**As a** Home Project developer **I want to** want quickly and clearly understand what went wrong with a build. The most interesting thing for me is errors that failed the build. But as for now, the build log is too verbose and it is pretty hard to navigate over it.

### Description
The most interesting thing for me is errors that failed the build. But as for now, the build log is too verbose and it is pretty hard to navigate over it.

### Acceptance Criteria
The maven builds log contains only important information for troubleshooting.

### Tasks 
- [ ] Configured maven build on CI
- [ ] Logs isn't too verbose

### PR DESCRIPTION
**Pull request by likeRewca**
dev
## ZenHub

* [Main ZenHub ticket](https://app.zenhub.com/workspaces/home-project-5f7b77ff8db73a001cb17009/issues/ita-social-projects/home/194)

## Code reviewers

- [ ] @kh0ma, @VadymSokorenko, @denis-litvinov, @DzigMS

### Second Level Review

- [ ] @github_username

## Summary of issue

Maven logs on CI have a lot of usless information.

## Summary of change

Add manev key -ntp into CI.yml

## Testing approach

Manual

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -19,8 +19,9 @@ jobs:
         env:
```
Comment (kh0ma) [inline .github/workflows/CI.yml:19]: Please do not forget that the same changes added here have to be mirrored to CD file.

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -19,8 +19,9 @@ jobs:
...
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+          DOCKER_INFO: false
```
Comment (kh0ma) [inline .github/workflows/CI.yml]: `API_TESTS_VERBOSE_LOGGING`

File: .github/workflows/CI.yml [hunk 3]
```diff
@@ -29,3 +30,4 @@ jobs:
           -Dpostgres.password=${{secrets.POSTGRES_PASSWORD}}
           -Dhome.application.admin.username=${{secrets.HOME_APPLICATION_ADMIN_USERNAME}}
           -Dhome.application.admin.password=${{secrets.HOME_APPLICATION_ADMIN_PASSWORD}}
+          -Ddocker.infrastructure=$DOCKER_INFO
```
Comment (kh0ma) [inline .github/workflows/CI.yml]: `-Dapi.tests.verbose.logging=$API_TESTS_VERBOSE_LOGGING`

File: home-api-tests/pom.xml [hunk 4]
```diff
@@ -21,6 +21,7 @@
         <docker.image.data.migration>home-data-migration:${revision}</docker.image.data.migration>
         <docker.image.application>home-application:${revision}</docker.image.application>
         <docker.image.postgres>postgres</docker.image.postgres>
+        <docker.infrastructure>${docker.infrastructure}</docker.infrastructure>
```
Comment (kh0ma) [inline home-api-tests/pom.xml]: `<api.tests.verbose.logging>true</api.tests.verbose.logging>`

File: home-api-tests/pom.xml [hunk 5]
```diff
@@ -32,6 +33,12 @@
         <jacoco.agent.internal.port>6300</jacoco.agent.internal.port>
     </properties>
     <profiles>
+        <profile>
```
Comment (kh0ma) [inline home-api-tests/pom.xml]: I don't think we need it. Please remove.

File: home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/utils/ApiClientUtil.java [hunk 6]
```diff
@@ -15,10 +15,13 @@ public static ApiClient getClient() {
         String applicationExternalPort = System.getProperty("home.application.external.port");
         String applicationAdminUsername = System.getProperty("home.application.admin.username");
         String applicationAdminPassword = System.getProperty("home.application.admin.password");
+        String verboseLogging = System.getProperty("verbose.tests.logging");
```
Comment (kh0ma) [inline home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/utils/ApiClientUtil.java]: `String verboseLogging = System.getProperty("verbose.tests.logging", "true");`

File: home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/utils/ApiClientUtil.java [hunk 7]
```diff
@@ -15,10 +15,13 @@ public static ApiClient getClient() {
...
         ApiClient client = new ApiClient();
         Logger logger = Logger.getLogger(ApiClient.class.getName());
-        client.getHttpClient()
-            .register(new LoggingFeature(logger, Level.INFO, LoggingFeature.Verbosity.PAYLOAD_ANY, 8192));
+        if (verboseLogging.equals("true")) {
```
Comment (kh0ma) [inline home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/utils/ApiClientUtil.java]: Use `java.lang.Boolean#parseBoolean`

**Review by kh0ma — CHANGES_REQUESTED** [review]

File: home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/utils/ApiClientUtil.java (hunk 7 above)
Comment (likeRewca) [reply home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/utils/ApiClientUtil.java]: Synced changes with Issue#197, unfortunately work with level logging was unsuccessful.

**Review by annaprok — APPROVED** [review]

**Review by VadymSokorenko — APPROVED** [review]

File: .github/workflows/CI.yml (hunk 1 above)
Comment (kh0ma) [reply .github/workflows/CI.yml:19]: It seems to me that this comment is still relevant.

**Review by kh0ma — CHANGES_REQUESTED** [review]
Please reflect the changes to CD file and we can move forward with it.

File: .github/workflows/CI.yml (hunk 1 above)
Comment (likeRewca) [reply .github/workflows/CI.yml:19]: I read and successfully forgot about it

**Review by denis-litvinov — APPROVED** [review]

**Review by kh0ma — APPROVED** [review]

**Pull request merged**